ReisTecnologia/abc
# Check table script

How to test:
Make sure to have a docker dynamodb local container running at localhost:8000.
Run createTable.js with node, it should create a table according to the description inside createTable.js
